man/MultiColony-class.Rd

```diff
@@ -7,8 +7,8 @@
 \alias{show,MultiColony-method}
 \alias{c,MultiColony-method}
 \alias{c,MultiColonyOrNULL-method}
-\alias{[,MultiColony,integerOrNumericOrLogical,ANY,ANY-method}
```

Comment:
This I don't know! It's different every time although the R file doesn't change  - and one of these is likes by CRAN and the other is not!

Reply:
@janaobsteter I saw that before a year ago or so - I think its down to how Roxygen is behaving. I think we just need to have a state where R CMD check is behaving well - try rerunning it until we get a pass. Bummer, I know.

Reply:
I've now copied the working version somewhere - and it's weird because I'm not updating roxygen day to day!